### ISSUE TITLE
Coal Not being mined

### ISSUE THREAD

**Issue by learningcoder3** [xJon/Tekkit-SMP#22]
### Was this bug reported before?

- [X] No, it was not.

### Please describe the issue you are experiencing

when using a red-matter or dark-matter pickaxe it is able to one-click mine every ore vein except coal, coal does not work with the feature.

**Comment by xJon** [comment]
Hi! Can you reproduce this in a creative singleplayer world? I'm unable to.

**Comment by learningcoder3** [comment]
yes, on a super flat i place coal ore and was unable to harvest with right click, I then placed Lapiz ore and was able to harvest with one click

**Comment by xJon** [comment]
Strange?

https://github.com/user-attachments/assets/a39362a0-f718-4071-909c-b669936694b2

**Comment by xJon** [comment]
Closing due to inactivity.
